pkg/odo/util/cmdutils.go

```diff
@@ -99,17 +100,29 @@ func PrintComponentInfo(client *occlient.Client, currentComponentName string, co
 
 	}
 
+	var storages storage.StorageList
```

Comment:
Since we don't need these two variables outside of the below if statement, I would suggest moving them inside.

Reply:
@mik-dass Thanks for review, updated